styles.css

```diff
@@ -135,11 +168,110 @@ li > * {
   gap: 10px;
 }
 
-.work {
+.work-direction {
   display: flex;
   justify-content: center;
   gap: 15px;
   padding-top: 25px;
   color: #a2a3ae;
   font-size: 20px;
 }
+
+/*
+  WORK
+*/
+
+.work {
+  margin-top: 20px;
+}
+
+#break {
+  display: block;
+}
+
+.all_projects {
+  display: grid;
+  grid-gap: 10px;
+}
+
+#work-title {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 20px;
+}
+
+.work h2 {
+  color: #9490ff;
+  font-size: 35px;
+  font-family: sans-serif, Roboto;
+  line-height: 52px;
+}
+
+.project {
+  display: grid;
+  grid-gap: 10px;
+  margin-top: 30px;
+  background: url();
+}
+
+.project-body {
+  display: grid;
+  grid-gap: 5px;
+}
+
+.project-body p {
+  font-family: sans-serif, Inter;
+  font-style: normal;
+  font-size: 20px;
+  line-height: 28px;
+  color: #dcdce8;
+}
```

Comment:
- [ ] Please make sure to center the elements as in the template. 
Currently, your cards have the image and button not centered, try fixing that to match the template.

![](https://i.imgur.com/PKIWd07.png)